Benchmark-Models/Rahman_MBS2016/observables_Rahman_MBS2016.tsv

```diff
@@ -0,0 +1,5 @@
+observableId	observableName	observableFormula	noiseFormula	observableTransformation	noiseDistribution
+observable_prevalence	prevalence	prevalence	noiseParameter1_observable_prevalence	lin	normal
+observable_immunity_high	immunity_high	infected_normal + treated_normal	noiseParameter1_observable_immunity_high	lin	normal
+observable_immunity_intermediate	immunity_intermediate	infected_moderate + treated_moderate	noiseParameter1_observable_immunity_intermediate	lin	normal
+observable_immunity_low	immunity_low	infected_weak + treated_weak	noiseParameter1_observable_immunity_low	lin	normal
```

Comment:
You only have measurements for one observable right? Why are there multiple observables here?